tests/embodied/robot/test_forward_kinematics.py

```diff
@@ -23,7 +23,7 @@
 @pytest.mark.parametrize("dtype", [torch.float32, torch.float64])
 def test_backward(batch_size: int, dtype: torch.dtype):
     robot = Robot.from_urdf_file(urdf_path, dtype)
-    selected_links = ["panda_link2", "panda_link5"]
+    selected_links = ["panda_link2", "panda_link5", "panda_virtual_ee_link"]
```

Comment:
Once we move new FK to labs we might need to recreate this test inside the labs FK package.